Update Set-CsTeamsVoiceApplicationsPolicy.md (#10866)
MicrosoftDocs/office-docs-powershell: teams/teams-ps/teams/Set-CsTeamsVoiceApplicationsPolicy.md

 
 [-AllowCallQueueWelcomeGreetingChange <Boolean>] 
 [-AllowCallQueueMusicOnHoldChange <Boolean>]
 [-AllowCallQueueOverflowSharedVoicemailGreetingChange <Boolean>]
 [-AllowCallQueueTimeoutSharedVoicemailGreetingChange <Boolean>]
 [-AllowCallQueueNoAgentSharedVoicemailGreetingChange <bool>]
 [-AllowCallQueueLanguageChange <Boolean>] 
 [-AllowCallQueueMembershipChange <Boolean>] 
 [-AllowCallQueueConferenceModeChange <Boolean>] 
 [-AllowCallQueueRoutingMethodChange <Boolean>] 
 [-AllowCallQueuePresenceBasedRoutingChange <Boolean>]
 [-AllowCallQueueOptOutChange <Boolean>] 
 [-AllowCallQueueOverflowRoutingChange <Boolean>]
 [-AllowCallQueueTimeoutRoutingChange <Boolean>] 
 [-AllowCallQueueNoAgentsRoutingChange <Boolean>]
 [-AllowCallQueueAgentOptChange <Boolean>] 
 
 [-CallQueueAgentMonitorMode <Disabled | Monitor | Whisper | Barge | Takeover>]
 [-CallQueueAgentMonitorNotificationMode <Disabled | Agent>]
 
 [-RealTimeAutoAttendantMetricsPermission <string>]
 [-RealTimeCallQueueMetricsPermission <string>]
 [-RealTimeAgentMetricsPermission <string>]
 
 [[-Identity] <String>]
 [-WhatIf] [-Confirm] [<CommonParameters>]
```

## DESCRIPTION

> `TeamsVoiceApplicationsPolicy` is used for **Supervisor Delegated Administration** which allows tenant admins to permit certain users to make changes to auto attendant and call queue configurations.

Reviewer: ```suggestion
`TeamsVoiceApplicationsPolicy` is used for **Supervisor Delegated Administration** which allows admins in the organization to permit certain users to make changes to auto attendant and call queue configurations.
```

```yaml
Type: Object
Parameter Sets: Dual
Aliases:
Applicable: Skype for Business Online

Required: False
Position: Named
Default value: Disabled
Accept pipeline input: False
Accept wildcard characters: False
```

### -CallQueueAgentMonitorNotificationMode

PARAMVALUE: Disabled | Agent

When set to `Disabled` (the default value), users affected by the policy won't be allowed to monitor agents during call sessions.

When set to `Agent`, users affected by the policy will be allowed to monitor agents during call sessions.

```yaml
Type: Object
Parameter Sets: Dual
Aliases:
Applicable: Skype for Business Online

Required: False
Position: Named
Default value: Disabled
Accept pipeline input: False
Accept wildcard characters: False
```

### -RealTimeAutoAttendantMetricsPermission

PARAMVALUE: None | Authorized | All

When set to `None` (the default value), users affected by the policy won't receive real-time metrics for auto attendants.

When set to `Authorized`, users affected by the policy will receive real-time metrics for auto attendants they are authorized for.
 

> When set to `All`, users affected by the policy will receive real-time metrics for all auto attendants in the tenant.

Reviewer: ```suggestion
When set to `All`, users affected by the policy will receive real-time metrics for all auto attendants in the organization.
```

```yaml
Type: Object
Parameter Sets: Dual
Aliases:
Applicable: Skype for Business Online

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```
 
### -RealTimeCallQueueMetricsPermission

PARAMVALUE: None | Authorized | All

When set to `None` (the default value), users affected by the policy won't receive real-time metrics for call queues.

When set to `Authorized`, users affected by the policy will receive real-time metrics for call queues they are authorized for.
 

> When set to `All`, users affected by the policy will receive real-time metrics for all call queues in the tenant.

Reviewer: ```suggestion
When set to `All`, users affected by the policy will receive real-time metrics for all call queues in the organization.
```

```yaml
Type: Object
Parameter Sets: Dual
Aliases:
Applicable: Skype for Business Online

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -RealTimeAgentMetricsPermission

PARAMVALUE: None | Authorized | All

When set to `None` (the default value), users affected by the policy won't receive real-time metrics for agents.

When set to `Authorized`, users affected by the policy will receive real-time metrics for agents who are members in the call queues they are authorized for.
 

> When set to `All`, users affected by the policy will receive real-time metrics for all agents in all queues call queues in the tenant.

Reviewer: ```suggestion
When set to `All`, users affected by the policy will receive real-time metrics for all agents in all call queues in the organization.
```
